README.md

```diff
@@ -2,23 +2,49 @@
 
 ## Requirements
 
-You must have configured a Miniconda installation before continuing.
-More information in the [Confluence Page](https://eden.esss.com.br/confluence/display/EDEN/EDEn+Quickstart).  
-All envs will be created in C:/ or D:/. Each env will be put on a directory with its name. Env
-names are usually numbers starting from 0.
+You must have a conda installation before continuing.
+See [Conda Intallation](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) 
 
 
-## Minimum folder structure for an environment
+## Installing
 
-- ```env_number/``` (Example: 1/)
-    - ```aa_conf/```
-        - ```aasimar10.conf```
+Run the following command in your conda base environment (that is, you must 'deactivate' from any current environment different from 'base')
+
+```
+conda install ww
+```
+
+### Change default workspaces location
+
+`ww` will create new workspaces on a default location, but that location must exist. If one does not, you can either create it or tell `ww` you want it to create workspaces on a different (and existent) location.
+
+#### Windows
+
+The default location for workspace creation is the root of the volume `W:`. 
+To change default workspace location you need to override the environment variable `WW_DEFAULT_VOLUMES`
+
+```
+setx WW_DEFAULT_VOLUMES=<VOLUME>
```

Comment:
```suggestion
setx WW_DEFAULT_VOLUMES <VOLUME>
set WW_DEFAULT_VOLUMES=<VOLUME>
```
`setx` do not use the `=` also suggest to execute normal `set` since `setx` do not change the current runing environment.

Reply:
Done